Xamarin.Forms.Platform.UAP/Shell/ShellSectionRenderer.cs

```diff
@@ -191,7 +175,49 @@ internal void NavigateToContent(ShellNavigationSource source, ShellContent shell
 			}
 		}
 
-		NavigationTransitionInfo GetTransitionInfo(ShellNavigationSource navSource)
+		protected void OnPopRequested(ShellNavigationSource source)
```

Comment:
Did you mean to make these virtual?

Reply:
omg, i triple checked the code inside the methods to be sure to avoid any mistake and then missed entirely the point of the pull request :( i need some sleep i think :)